test/test_parse.c

```diff
@@ -6912,6 +6912,27 @@ START_TEST(test_issue991)
 }
 END_TEST
 
+START_TEST(test_issue1038)
+{
+   opt_set_int(OPT_RELAXED, 1);
+   set_standard(STD_08);
+
+   input_from_file(TESTDIR "/parse/issue1038.vhd");
+
+   const error_t expect[] = {
+      { 14, "aliased name is not static" },
```

Comment:
I think this is a bug too, a unit name should be a static name. Probably need to add `T_UNIT_DECL` to the list in `sem_static_name`.

Reply:
I've added an additional commit for it. Or would you like it to be an individual PR?

Reply:
It's fine in this one.